app/src/main/java/com/example/todokotlinandroid/ComposeColors.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.todokotlinandroid
+
+import androidx.compose.ui.graphics.Color
+
+val GreenColor = Color(0x63BF42)
```

Comment:
All colors have to have alpha channel
otherwise they will not show on screen. 
You're missing FF after 0x. So for example GreenColor should be Color(0xFF63BF42)